docs/docs/concepts/what-is-core.md

```diff
@@ -0,0 +1,14 @@
+---
+myst:
+  html_meta:
+    "description": "kitconcept.core how-to guides"
+    "property=og:description": "kitconcept.core how-to guides"
+    "property=og:title": "kitconcept.core how-to guides"
+    "keywords": "Plone, kitconcept.core, how-to, guides"
+---
+
+# What is `kitconcept.core`?
+
+It's a backend and frontend setup that serves as baseline and foundation for all kitconcept GmbH distributions built on top of Plone.
+
+It includes common features, add-ons, configuration, best practices, and shared components that are used across multiple projects.
```

Comment:
That is also true of VLT and even Volto core. The thing that makes kitconcept-core the right place to put it is if it's something that we want for all kitconcept projects (both intranet and public website) but that is not the right fit for everyone in the Plone community.

Reply:
I don't know if I get your point. I wrote this quickly, as a placeholder, at least to have something. I hope that we all can complete it.